backend/src-common/src/main/java/org/eclipse/sw360/mail/MailUtil.java

```diff
@@ -113,6 +118,19 @@ public void sendMail(Set<String> recipients, String excludedRecipient, String su
         sendMail(recipients, excludedRecipient, subjectNameInPropertiesFile, textNameInPropertiesFile, notificationClass, roleName, true, textParameters);
     }
 
+    public void sendMailAsync(MimeMessage message) {
+        ExecutorService mailExecutor = Executors.newSingleThreadExecutor();
```

Comment:
Instead of creating a new Executor every time it might be better to have static or persistent one.

Reply:
changed to static